packages/dataviews/src/components/dataviews-view-config/style.scss

```diff
@@ -0,0 +1,43 @@
+.dataviews-view-config {
+	width: 363px;
+	container-type: inline-size;
+	padding: $grid-unit-20;
+}
+.dataviews-view-config__sort-direction .components-toggle-group-control-option-base {
+	text-transform: uppercase;
+}   
+
+.dataviews-settings-section__title.dataviews-settings-section__title {
+	line-height: $grid-unit-30;
+    font-size: 15px;
```

Comment:
Not something to address here, but it would be great to update the `Text` / `Heading` components to support type styles so that we can avoid this kind of css. cc @tyxla.

Reply:
I believe both the `Text` and the `Heading` components accept a `size` prop that you can use:

```
<Heading size={ 18 } />My heading</Heading>
<Text size={ 13 } />My text</Text>
```

Now, I can argue whether doing this is a good idea. Specifying an arbitrary size can work against our efforts to bring consistency across the editor. Ideally, you would use one of the available `level` values of `Heading` (1 to 6), and `Text` would be consistent across the editor. 

cc @WordPress/gutenberg-components in case anyone else has further thoughts.

Reply:
What @jameskoster said 💯